workers/ohsome_quality_analyst/cli.py

```diff
@@ -105,20 +106,20 @@ def create_indicator(
 ):
     """Create an Indicator.
 
-    Write a GeoJSON with the result to disk if an outfile is specified.
+    Output is a GeoJSON Feature or FeatureCollection with the indicator results.
+    Write to disk if the `outfile` option is specified.
```

Comment:
see other comment about the `outfile` parameter. If the code stays at it is, it should be stated that the output **only** goes to disk, if `outfile` is specified.

Reply:
See answer to above comment.